modules/imap/hm-imap.php

```diff
@@ -1038,6 +1039,9 @@ public function get_message_list($uids, $raw=false, $include_content_body = fals
                                 }
                             }
                         }
+                        elseif ($vals[$i] == 'invite.ics' ) {
```

Comment:
Seems like a random hard-coded name to check. If you want to check for invitation, maybe use content type or other more-reliable method? What if the file is called invitation.ics? Maybe check for .ics extension?